charts/steward/templates/clusterrole-run-controller.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: steward-run-controller
+  labels:
+    {{- include "steward.labels" . | nindent 4 }}
+rules:
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["rolebindings"]
+  verbs: ["get","list","create"]
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["clusterroles"]
+  verbs: ["bind","get"]
+  resourceNames:
+  {{- toYaml .Values.runController.mappableClusterRoles | nindent 2 }}
```

Comment:
I would expect an indent of 4.

In general using inline json is better than counting spaces.

Reply:
I think 2 is sufficient. The list items need only the same or longer ident as the key.
---
foo:
- bar
- baz
---
this is a valid list in a map like
{"foo": ["bar","baz"]}

Reply:
The syntax is correct, yes. My yaml linting tool would give an error, however.
To understand that the syntax is correct you have to know that the values has to be a list.

Reply:
I agree to either indent by 4 spaces or use a JSON list.